packages/apollo-codegen-core/src/serializeToJSON.ts

```diff
@@ -51,13 +53,15 @@ export function serializeAST(ast: any, space?: string) {
   );
 }
 
-function serializeType(type: GraphQLType, options: serializeOptions) {
+function serializeType(type: GraphQLType, options?: serializeOptions) {
```

Comment:
Making `options` optional here isn't really related to the work in this PR, but TS was throwing a warning in the existing test file I had to modify. Looking over the code, `options` is truly optional, so let's make that evident.